src/DETHRACE/constants.h

```diff
@@ -485,15 +485,13 @@ enum {
 
 #define NONCAR_UNUSED_SLOTS 5
 
-#define ONEEIGHTTWO 182.0444444444444
-
-#define d180_OVER_PI 57.29577951308232 // (180 / PI)
-
 #define DR_PI 3.141592653589793
+#define DR_PI_OVER_2 1.570796326794897
+#define DR_3PI_OVER_2 4.71238898038469
 
 #define TIME_CONV_THING 0.0005f
 
-#define OPPONENT_COUNT 5
+#define OPPONENT_COUNT 0
```

Comment:
Don't forget about this.

Reply:
ah crap :P

Reply:
I should make this a `--opponents` flag so I don't keep making the same dang mistake!

Reply:
Name it `--baby-mode` 🤣

Reply:
But you suggestion is probably better. It can accept an integer to test a zillion opponents.